mesa/time.py

```diff
@@ -39,12 +39,24 @@
 
 
 class BaseScheduler:
-    """Simplest scheduler; activates agents one at a time, in the order
-    they were added.
-
-    Assumes that each agent added has a *step* method which takes no arguments.
-
-    (This is explicitly meant to replicate the scheduler in MASON).
+    """
+    A simple scheduler that activates agents one at a time, in the order they were added.
+
+    This scheduler is designed to replicate the behavior of the scheduler in MASON, a multi-agent simulation toolkit.
+    It assumes that each agent added has a `step` method which takes no arguments and executes the agent's actions.
+
+    Attributes:
```

Comment:
It currently renders without formatting: https://mesa--1873.org.readthedocs.build/en/1873/apis/time.html.

It seems that you could use info fields: https://stackoverflow.com/questions/8649105/how-to-show-instance-attributes-in-sphinx-doc, which renders better. This is not a blocker to this PR getting merged. Only if you have the time for it.

Reply:
I just added list markers (`- `), that seems to do the trick